docs/source/install.rst

```diff
@@ -3,8 +3,8 @@
 Installation
 ============
 
-NEoST is a Python software package that performs neutron star equation 
-of state inference process. it is an open source package that was 
+NEoST is a Python software package that performs neutron star equation
+of state inference process. it is an open source package that was
```

Comment:
Capital I at start of 2nd sentence

Reply:
Done, also changed "was and is" to "is". It's not really important that NEoST was available on github in the past :D